<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UusiToimistoon</class>
 <widget class="QWidget" name="UusiToimistoon">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>502</width>
    <height>483</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Ryhm√§</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Kirjanpidon tuote</string>
       </property>
       <property name="buddy">
        <cstring>tuoteList</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QListWidget" name="tuoteList"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Tilitoimisto</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="toimistoLabel">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="ryhmaLabel">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="harjoitusCheck">
     <property name="text">
      <string>Harjoituskirjanpito</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>175</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tuoteList</tabstop>
  <tabstop>harjoitusCheck</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
